.github/workflows/check-layers.yml

```diff
@@ -13,8 +13,17 @@ jobs:
       matrix:
         host-os:    ["ubuntu-latest"]
```

Comment:
Since `ubuntu-latest` is the only platform used, you can remove it from the matrix and just use
```
runs-on: "ubuntu-latest"
```